ocs_ci/utility/utils.py

```diff
@@ -624,7 +624,12 @@ def exec_cmd(
     log.info(f"Executing command: {masked_cmd}")
     if isinstance(cmd, str) and not kwargs.get("shell"):
         cmd = shlex.split(cmd)
-    if cluster_config and cmd[0] == "oc" and "--kubeconfig" not in cmd:
+    if (
```

Comment:
This doesn't fix the issue for multicluster execution, as we need to maintain the list of those subcommands and check if it's in cmd, if so, to increase the position where we will insert kubeconfig on line 634 and 635.

Reply:
ack, I will fix this